amip_interp/amip_interp.F90

```diff
@@ -544,7 +544,11 @@ subroutine get_amip_sst (Time, Interp, sst, err_msg, lon_model, lat_model)
                              lon_model, lat_model, interp_method="bilinear" )
 
     if (use_mpp_io) then
-       the_file_exists = fms_io_file_exists(ncfilename)
+            !! USE_MPP_IO_WARNING
+            call mpp_error ('amip_interp_mod', &
+             'MPP_IO is no longersupported.  Please remove from namelist',&
+              WARNING)
```

Comment:
Should we take off the gloves and make the WARNINGs FATALs, given the tendency to ignore them?

Reply:
I thought that was supposed to be in the subsequent release?

Reply:
FATAL will be in the release after we delete mpp_io.

Reply:
Ah, gotcha.